### PR TITLE
New: Paged Grid

### PR DESCRIPTION
**Pull request by vinteo**
image

### CODE REVIEW

File: src/components/adslotUi/PagedGridComponent.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+                </GridCell>
+              )}
+            </GridRow>
+          )}
+          <Empty collection={items} icon={emptyIcon} svgSymbol={emptySvgSymbol} text={emptyMessage} />
+        </Grid>
+        {totalPages > 1 ?
+          <div className="pagedgrid-component-pagination">
+            <span className="pagedgrid-component-pagination-info">
+              {(activePage - 1) * perPage + 1}–{Math.min(activePage * perPage, items.length)} of {items.length}
```
Comment (OndrejCholeva) [inline src/components/adslotUi/PagedGridComponent.js]: I am not sure about correctness of this - check your screenshot `1-2 of 3 < 1, 2 >` (of 3 when there are 2 pages doesn't make sense).

**Comment by omgaz** [comment]
LGTM

**Comment by vinteo** [comment]
@omgaz merge?

**Comment by omgaz** [comment]
Merging :ramen:

**Pull request merged**